java/src/main/java/com/xxAMIDOxx/xxSTACKSxx/menu/mappers/RequestToCommandMapper.java

```diff
@@ -8,11 +8,16 @@
 import com.xxAMIDOxx.xxSTACKSxx.menu.commands.CreateItemCommand;
 import com.xxAMIDOxx.xxSTACKSxx.menu.commands.CreateMenuCommand;
 import com.xxAMIDOxx.xxSTACKSxx.menu.commands.UpdateMenuCommand;
+import com.xxAMIDOxx.xxSTACKSxx.menu.commands.DeleteMenuCommand;
+import com.xxAMIDOxx.xxSTACKSxx.menu.commands.OperationCode;
 
 import java.util.UUID;
 
 public class RequestToCommandMapper {
 
+    private RequestToCommandMapper() {
```

Comment:
Is this needed? All methods should be static

Reply:
sonar lint was complaining hence added